lib/tftp/src/tftp.erl

```diff
@@ -590,7 +427,7 @@ matching regexp is found.
 -spec write_file(RemoteFilename, LocalFilename, Options) ->
         {ok, LastCallbackState} | {error, Reason} when
     RemoteFilename    :: file:filename(),
-    LocalFilename     :: file:filename_all(),
+    LocalFilename     :: file:filename_all() | binary,
```

Comment:
Is the atom `binary` really correct here, or should that be `binary()`?

Reply:
Yes it is according to the code. file:filename_all  already covers binary().